lib/serverroutes.js

```diff
@@ -345,9 +345,8 @@ module.exports = function(app, saveSecurityConfig, getSecurityConfig) {
       securityWasEnabled = true
 
       function addUser(req, res, securityStrategy, config) {
-        var config
         if (!config) {
-          config = getSecurityConfig(app)
+          config = app.securityStrategy.getConfiguration()
```

Comment:
I always consider function parameters immutable, so I preferred the original in that respect. But that was overloading the name.